src/components/molecules/TaskCandidatesSection/TaskCandidatesSection.tsx

```diff
@@ -67,15 +74,18 @@ const TaskCandidatesSection: FC<TaskCandidatesSectionProps> = ({
           name,
           status,
           price,
-          delete_button: id,
+          delete_button: vendor.id,
         }
       }),
     [candidates, getCandidateStatus]
   )
 
-  const handleDelete = useCallback((candidateId: string) => {
-    // TODO: remove candidate with id: candidateId from this task
-  }, [])
+  const handleDelete = useCallback(
+    (vendor_id: string) => {
+      deleteAssignmentVendor({ data: [{ vendor_id }] })
```

Comment:
[blocker]
Deleting error/success message is missing

Reply:
yes, we use showNotification. we don't have the messages, but make them all similar.

Reply:
Done, please check